include/xtensor-io/xblosc.hpp

```diff
@@ -163,16 +164,25 @@ namespace xt
     {
         const char* name;
         const char* version;
+        bool big_endian;
         int clevel;
         int doshuffle;
 
         xblosc_config()
             : name("blosc")
             , version(BLOSC_VERSION_STRING)
+            , big_endian(false)
             , clevel(5)
             , doshuffle(1)
         {
         }
+
+        template <class T>
+        void write(T& j) const
```

Comment:
Maybe the method should be renamed `write_to`? Or even `write_to_json` to be even more explicit?

Reply:
It it true that we currently use this method to write to a JSON object, but it can actually be any object that supports key-value assignment. So `write_to_json` might not be a good name in general.
I'm okay with changing to `write_to`.